manifests/base/applicationset/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://raw.githubusercontent.com/argoproj/applicationset/release-0.3.0/manifests/install.yaml
```

Comment:
I believe this should point to `https://raw.githubusercontent.com/argoproj/applicationset/master/manifests/install.yaml`

I have included my reasoning here: https://github.com/argoproj/argo-cd/issues/7351#issuecomment-1020282879